stream-transformation-test/stream-transformation-performance-test/src/test/java/uk/gov/justice/framework/tools/transformation/GenerateTestData.java

```diff
@@ -18,17 +18,25 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * This class by default creates 1000 events with 100 streams with 10 events per stream These can be
```

Comment:
you could potentially use the @link #variableName to point to the 100, 10 events so it's clear where to adjust.